R/service-navigation.R

```diff
@@ -0,0 +1,138 @@
+#' Service navigation
+#'
+#' @description
+#' Service navigation component consistent with the
+#' [GDS service navigation](https://design-system.service.gov.uk/components/service-navigation/)
+#' (pre-June 2025).
+#'
+#' @param links A vector of actionLinks to be added to the service navigation.
```

Comment:
Really like this, gives flexibility alongside simplicity. Think this is my preferred way to do the function!